web/router.ex

```diff
@@ -45,18 +45,18 @@ defmodule Pairmotron.Router do
   scope "/", Pairmotron do
     pipe_through [:browser, :authenticate] # Use the default browser stack
 
-    get "/pairs", PairController, :index
```

Comment:
I don't like having `/pairs` redirect to one of the user's group pair pages. I think its surprising in the best case. It would be less surprising and more useful to have `/pairs` include the pairs for each of the user's groups for that time period. I'm not going to implement that unless we both think its a good idea.